src/controllers/routes.js

```diff
@@ -8,6 +8,9 @@ export const routes = {
   movies: {
     getAllMovies: Config.API_BASE_URL + 'discover/movie?api_key=' + Config.API_KEY,
   },
+  moviesByPage: {
+    getMoviesByPage: Config.API_BASE_URL + 'discover/movie?api_key=' + Config.API_KEY + '&page=',
+  },
```

Comment:
The idea behind having a networkService is not to add for every route the API key or API base as in most of the cases those will repeats.
Please, find a way that do not need to add those things to routes and only on the network service.